lwip/sys_arch.c

```diff
@@ -210,7 +210,7 @@ unsigned long ulReturn;
 	{
 		configASSERT( xInsideISR == ( portBASE_TYPE ) 0 );
 
-		if( pdTRUE == xQueueReceive( *pxMailBox, &( *ppvBuffer ), ulTimeOut/ portTICK_RATE_MS ) )
+		if( pdTRUE == xQueueReceive( *pxMailBox, &( *ppvBuffer ), ulTimeOut ) )

```

Comment:
ulTimeout is supposed to be in ms. If so, it should not be divided by portTICK_RATE_MS